src/main/java/com/prgrms/vouchermanagement/voucher/domain/FixedAmountVoucher.java

```diff
@@ -0,0 +1,34 @@
+package com.prgrms.vouchermanagement.voucher.domain;
+
+import java.util.UUID;
+
+public class FixedAmountVoucher implements Voucher {
+	private final UUID voucherId;
+	private final long amount;
+
+	public FixedAmountVoucher(UUID voucherId, long amount) {
+		if (amount <= 0)
+			throw new IllegalArgumentException("FixedAmountVoucher - Amount should be positive");
```

Comment:
```suggestion
		if (amount <= 0) {
			throw new IllegalArgumentException("FixedAmountVoucher - Amount should be positive");
		}
```

현재는 많은 수의 코드가 있는 것은 아니기 때문에 티가 안날 수도 있지만 indent가 들어가는 곳이 있다면 중괄호로 구분하는 것을 권장합니다.